This patch adds Solaris as a supported OS.  This patch has been
submitted upstream but will not be accepted until Solaris 12 is
released publicly AND a Puppetlabs customer requests its support for
Puppet Enterprise.

--- puppetlabs-rabbitmq-5.3.1/metadata.json.orig   2015-11-17 09:49:31.060947205 -0800
+++ puppetlabs-rabbitmq-5.3.1/metadata.json       2015-11-17 09:50:42.946972161 -0800
@@ -40,6 +40,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "11.3",
+        "12.0"
+      ]
     }
   ],
   "requirements": [
